### PR TITLE
Docbase

### PR DESCRIPTION
**Pull request by XiaSun-Atmos**
Grammar check and minor changes

### CODE REVIEW

File: docs/source/2018Michael.rst [hunk 1]
```diff
@@ -57,10 +57,10 @@ Hurricane Track and Intensity
   :width: 400
   :align: center
 
-  Hurricane tracks from GFSv16beta (red dashline), GFSv15p2 (blue dash line), and BestTrack (black dash line). The dots are color coded with the vortex maximum 10-m wind speed (WS, kt). 
+  Hurricane tracks from GFSv16beta (red dash line), GFSv15p2 (blue dash line), and BestTrack (black dash line). The dots are color coded with the vortex maximum 10-m wind speed (WS, kt).
```
Comment (climbfuji) [inline docs/source/2018Michael.rst]: I think it is "dashed" line - but please double check.

File: docs/source/2019Haloween.rst [hunk 2]
```diff
@@ -37,14 +37,14 @@ Case Results
...
-500mb Geopotentail Height and Aboslute Vorticity Map
+500 mb Geopotential Height and Absolute Vorticity Map
 ======================================================
 
 .. figure:: images/500mb_2019110100_150s.png
   :width: 1200
   :align: center
 
-  500 hPa geopotential heights (Dam), absolute vorticity ( 10 :sup:`-5`/s), and winds (m/s)
+  500 hPa geopotential heights (Dam), absolute vorticity (10 :sup:`-5`/s), and winds (m/s)
```
Comment (climbfuji) [inline docs/source/2019Haloween.rst]: What does "Dam" stand for?

File: docs/source/auto_examples/TC_WS_TimeSeries.py [hunk 3]
```diff
@@ -64,6 +64,6 @@
 plt.savefig('tracker_ws_Barry_ufsv1.png')
 
 # %%
-# .. figure:: images/thumb/sphx_glr_tracker_ws_Barry_thumb.png
+# .. figure:: /auto_examples/images/thumb/sphx_glr_tracker_ws_Barry_thumb.png
```
Comment (climbfuji) [inline docs/source/auto_examples/TC_WS_TimeSeries.py]: Absolute paths?

**Review by climbfuji — CHANGES_REQUESTED** [review]
This is great, thank you! Just a few minor comments.

File: docs/source/2019Haloween.rst (hunk 2 above)
Comment (XiaSun-Atmos) [reply docs/source/2019Haloween.rst]: dam=10m, Dam is changed to dam

File: docs/source/auto_examples/TC_WS_TimeSeries.py (hunk 3 above)
Comment (XiaSun-Atmos) [reply docs/source/auto_examples/TC_WS_TimeSeries.py]: changed back to the relative paths

File: docs/source/2018Michael.rst (hunk 1 above)
Comment (XiaSun-Atmos) [reply docs/source/2018Michael.rst]: Done

Comment (climbfuji) [reply docs/source/2018Michael.rst]: I still think it is "dashed" and not "dash" ;-)

File: docs/source/auto_examples/TC_WS_TimeSeries.py (hunk 3 above)
Comment (XiaSun-Atmos) [reply docs/source/auto_examples/TC_WS_TimeSeries.py]: Please check the latest commit below. Can I modify earlier commits? I have to figure out how.

File: docs/source/2018Michael.rst [hunk 4]
```diff
@@ -57,7 +57,7 @@ Hurricane Track and Intensity
   :width: 400
   :align: center
 
-  Hurricane tracks from GFSv16beta (red dash line), GFSv15p2 (blue dash line), and BestTrack (black dash line). The dots are color coded with the vortex maximum 10-m wind speed (WS, kt). 
+  Hurricane tracks from GFSv16beta (red dashed line), GFSv15p2 (blue dashed line), and BestTrack (black dashed line). The dots are color coded with the vortex maximum 10-m wind speed (WS, kt).
```
Comment (XiaSun-Atmos) [inline docs/source/2018Michael.rst:60]: See here @climbfuji

**Review by climbfuji — APPROVED** [review]

**Pull request merged**